docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Stage 1: Build the application
+FROM node:18 AS build
```

Comment:
is there a reason we're not using the node images in our ECR like other projects. IE:
`FROM 169942020521.dkr.ecr.eu-west-2.amazonaws.com/base/node-18:18-alpine-builder`
`FROM 169942020521.dkr.ecr.eu-west-2.amazonaws.com/base/node-18:latest`

Got those from here: https://eu-west-2.console.aws.amazon.com/ecr/repositories/private/169942020521/base/node-18?region=eu-west-2